examples/example.krun

```diff
@@ -73,3 +73,7 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+AMPERF_RATIO_BOUNDS = 0.97, 1.03
+AMPERF_BUSY_THRESHOLD = 3.4 * 1024 * 1024 * 1024  # 3.6 GHz
```

Comment:
Sarah's pointing out that the comment says 3.*6*Ghz but the constant is 3.*4*Ghz.